### ISSUE TITLE
Suggestion: stats of time saved

### ISSUE THREAD

**Issue by SpitFire-666** [WofWca/jumpcutter#69]
I was hoping I could easily find/see time saved (eg minutes/seconds) when watching a video. Perhaps I'm not reading the UI properly?

It would be even cooler if it had long-term stats, like total time saved since installation of the addon (ublock has this).

Thanks, really loving this addon!

**Comment by WofWca** [comment]
Yeah you're right, it shows time saved stats based on the last 10 minutes of playback, with exponential decay (i.e. the older the data the less weight it has).
You can change the "time saved everaging method" in options to "all-time average", but it tracks it on a per-page basis for now. Do you think it should be the default? I thought the playback rate representation instead of absolute values is more relatable.

Total time saved on all pages is cool, I thought of it, but I was a little afraid that it's not a super good measure since you can change settings on the fly and it depends on what video you're watching. But yeah, I think it needs to be implemented. Perhaps we need to not take into account the time when "volume Threshold" is 0 and that's it.
Also need to think where to put this info, because you want to know both all-time stats and last-n-minutes stats.

**Comment by WofWca** [comment]
Also #69.
Nice.

**Comment by SpitFire-666** [comment]
Awesome! personally, I haven't bothered adjusting the settings at all, happy with the default options. 

I'm sure you could go full nerd about this, but nothing wrong with keeping it simple, eg
- Time saved (this tab): xxxx
- Total Time Saved since install (2022-05-31): xxxxx
- Bonus points: percentage of time saved

uBlock's stats:

![image](https://user-images.githubusercontent.com/38451588/172841361-bd58c4c6-9c42-4670-a4e2-fbbd09711c98.png)

**Comment by WofWca** [comment]
> Also need to think where to put this info

See the mockup in #110. It would fit into that design.

**Comment by WofWca** [comment]
Related: https://github.com/vantezzen/skip-silence/issues/139
